frontend/components/AddSite/index.js

```diff
@@ -6,21 +6,25 @@ import { connect } from 'react-redux';
 import { USER } from '../../propTypes';
 import TemplateSiteList from './TemplateSiteList';
 import AddRepoSiteForm from './AddRepoSiteForm';
+import AlertBanner from '../alertBanner';
 import { availableEngines } from '../SelectSiteEngine';
 import siteActions from '../../actions/siteActions';
 import addNewSiteFieldsActions from '../../actions/addNewSiteFieldsActions';
 
 const propTypes = {
+  alert: PropTypes.object,
```

Comment:
Could this be filled out more fully, perhaps as a new `PropType.shape` in our custom `propTypes.js`?